airflow/api_fastapi/parameters.py

```diff
@@ -171,7 +180,10 @@ def to_orm(self, select: Select) -> Select:
                 f"the attribute does not exist on the model",
             )
 
-        column = self.attr_mapping.get(lstriped_orderby, None) or getattr(DagModel, lstriped_orderby)
+        if self.to_replace:
+            lstriped_orderby = self.to_replace.get(lstriped_orderby, lstriped_orderby)
+
+        column = self.attr_mapping.get(lstriped_orderby, None) or getattr(self.model, lstriped_orderby)
```

Comment:
We use it only here, and that's the attribute ? I would say.

Reply:
Do you mean the model?

Reply:
yes the model